### DISCUSSION TITLE
Question: Support for hierarchical partition keys?

### DISCUSSION THREAD

**Discussion by sven5** [IEvangelist/azure-cosmos-dotnet-repository#333] [Ideas]
This is just a question: Are hierarchical partition keys currently supported? I cannot find how to configure them.
Reference: https://learn.microsoft.com/en-us/azure/cosmos-db/hierarchical-partition-keys
Thanks

**Comment by IEvangelist** [comment]
> This is just a question: Are hierarchical partition keys currently supported? I cannot find how to configure them. Reference: https://learn.microsoft.com/en-us/azure/cosmos-db/hierarchical-partition-keys Thanks

Unfortunately, not yet. This might be something for us to consider implementing though if there is enough interest.

**Comment by dcuccia** [comment]
I would like to take advantage of this feature, too

**Comment by Qualizorg** [comment]
Definitely would sign up for this!
